flight/Modules/Airspeed/airspeed.c

```diff
@@ -104,10 +104,10 @@ static int8_t airspeedADCPin=-1;
 // Private functions
 static void airspeedTask(void *parameters);
 void baro_airspeedGet(BaroAirspeedData *baroAirspeedData, uint32_t *lastSysTime, uint8_t airspeedSensorType, int8_t airspeedADCPin);
-static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev);
+static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev, void *ctx, void *obj, int len);
```

Comment:
It's not the 4 parameters that bother me so much as the default call with lots of ```NULL```s and ```0```s. That's not a pattern I'm particularly fond of, and I think we're seeing how it can be creaky with the HAL refactoring.

So if a struct pointer isn't a good approach, is there another idea for making that a little more transparent?